draft-ietf-masque-h3-datagram.md

```diff
@@ -623,69 +339,24 @@ recent draft version from the intersection set. This ensures that both
 endpoints negotiate the same draft version.
 
 
-# The Sec-Use-Datagram-Contexts HTTP Header {#context-hdr}
+# The Sec-Capsule-Protocol Header
+
+Definitions of new HTTP Methods or of new HTTP Upgrade Tokens that use the
+Capsule Protocol MAY use the Sec-Capsule-Protocol header to simplify
+intermediary processing.
 
-Endpoints indicate their support for datagram contexts by sending the
-Sec-Use-Datagram-Contexts header with a value of ?1. If the header is missing
-or has a value different from ?1, that indicates that its sender does not wish
-to use datagram contexts. Endpoints that wish to use datagram contexts SHALL
-send the Sec-Use-Datagram-Contexts header with a value of ?1 on requests and
-responses that use the capsule protocol.
+Endpoints indicate that the Capsule Protocol is in use on the data stream by
+sending the Sec-Capsule-Protocol header with a value of ?1. Intermediaries MAY
+use this header to allow processing of HTTP Datagrams for unknown HTTP methods
+or unknown HTTP Upgrade Tokens.
```

Comment:
What if this appears on a response that doesn't have a 2xx status?  What about 1xx?

Reply:
Clarified that Sec-Capsule-Protocol is only allowed on 2xx